infra/discovery/src/lib/search.js

```diff
@@ -140,14 +173,25 @@ class Listing {
   /**
    * Searches for listings.
    * @param {string} query - The search query.
+   * @param {string} sort - The target value to sort on (can be object notation)
+   * @param {string} order - The order of the sort
    * @param {array} filters - Array of filter objects
    * @param {integer} numberOfItems - number of items to display per page
    * @param {integer} offset - what page to return results from
-   * @param {boolean} idsOnly - only returns listing Ids vs listing object.
    * @throws Throws an error if the search operation failed.
    * @returns A list of listings (can be empty).
    */
-  static async search(query, filters, numberOfItems, offset, idsOnly) {
+  static async search(query, sort, order, filters, numberOfItems, offset) {
+    const currencies = [
+      'token-ETH',
+      'token-DAI',
+      'fiat-JPY',
```

Comment:
What about CNY and SGD ?

Let's not do it in this PR, but it seems there are many places where we define more or less the same currency list. Ideally we should have a single constant that we import everywhere... :)

Reply:
You're right I should add those currencies here, this was a temporary array as I was waiting for feedback on moving to a single source of truth for the currrency list. Happy to add them now and we can defer the currency list decision to another PR.